<launch>
	<!--NatashaApi-->
	<node name="natashaApi" pkg="webots_natasha" type="NatashaApi.py" output="screen" >    </node>

	<!-- Movement -->
	<include file="$(find humanoid_walking)/launch/natasha_walking.launch"></include>
	<node name="movementPredefined" pkg="movement_predefined" type="predefined_movement.py" output="screen" >    </node>

	<!-- Simulation Webots
  	<include file="$(find webots_natasha)/launch/start_simulation.launch"/>-->
    
	<!-- Arduino
	<node name="arduino" pkg="rosserial_python" type="serial_node.py" output="screen">
		<param name="port" value="/dev/ttyACM0" />
		<param name="baud" value="57600" />
	</node>-->

	<!-- Actions -->
	<node name="brainAttacker" pkg="state_machine" type="brainAttacker.py" output="screen" >    </node>

	<!-- Vision 
	<include file="$(find object_finder)/launch/test_mode.launch"></include>>-->
	<!--<node name="Behaviour_test" pkg="ballsim" type="Behaviour_test.py" output="screen">    </node>-->
	<node name="vision" pkg="object_finder" type="connecting_and_showing_current_frame_robocup.py" output="screen" >   </node>

</launch>
